### DISCUSSION TITLE
2021 Season 4

### DISCUSSION THREAD

**Discussion by alalonde** [MetaFam/TheGame#995] [General]
We've had some very active builders over the last few months, and it's showed! We knocked off the majority of the highest-priority features for MyMeta for version 1, including:

- The long-awaited [Dashboard](https://my.metagame.wtf/dashboard)! If you have a MyMeta profile, you can view detailed information about your XP, the SEED market, upcoming MetaGame events, the leaderboard, and the most recent text / audio / video content in the wider MetaFam universe. This was a huge feature with major contributions from our rock-star builders @vidvidvid, @Udit-takkar, @luxumbra 
- Links from the MegaMenu now open within MyMeta (mostly, anyway). You can now buy SEEDs, trade, access MetaGame content, and a whole lot more without even leaving MyMeta. This is a hugely important step towards the vision of the [MetaOS](https://github.com/MetaFam/TheGame/issues/913).
- You can now select your primary role (e.g. builder, designer, innkeeper) during the onboarding wizard in MyMeta, which makes the -role-select channel in Discord unnecessary and moves us closer to having the entire onboarding flow in MyMeta.

## Cheers 🍻 to our new contributors for this release! 
* @polimyl made their first contribution in https://github.com/MetaFam/TheGame/pull/841
* @jaburx made their first contribution in https://github.com/MetaFam/TheGame/pull/711
* @marswong made their first contribution in https://github.com/MetaFam/TheGame/pull/914
* @firattale made their first contribution in https://github.com/MetaFam/TheGame/pull/963

**Full Changelog**: https://github.com/MetaFam/TheGame/compare/v0.1...v0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/MetaFam/TheGame/releases/tag/v0.2'>2021 Season 4</a>.</em>

**Comment by luxumbra** [comment]
Thanks for putting this update here @alalonde and for your part in merging all these updates!! Great to see the progress we made in recent months. Long may it continue. 🚀
